extensions.lua

```diff
@@ -203,16 +247,32 @@ local function apply_config(conf)
         rawset(_G, fun_name, nil)
     end
 
+    if cartridge_args.http_enabled == true then
+        local to_remove = {}
+        for path, _ in pairs(vars.http_exports) do
+            for i, route in ipairs(httpd.routes) do
+                if path:match(route.match) or string.match(path .. '/', route.match) then
+                    table.insert(to_remove, i)
+                end
+            end
+        end
+        remove_old_routes(httpd.routes, to_remove)
+    end
```

Comment:
It's ambiguous

Reply:
I think that's OK. I've tried to rewrite this piece of code but it led to more complex constructions.